poh/src/poh_recorder.rs

```diff
@@ -140,7 +140,7 @@ pub struct RecordTransactionsSummary {
     pub starting_transaction_index: Option<usize>,
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
Do we really need `Debug` on all of this? I can't imagine anyone has ever used it for the massive scheduling and bank structures. I'd sooner remove it from all of that than let it spread further, wdyt?

Reply:
hmm, i think `Debug` is handy for quick debugging and actually required by `assert_matches!()`...